pyproject.toml

```diff
@@ -55,7 +55,7 @@ isort .\
  && flake8 openapi_python_client\
  && safety check --bare\
  && mypy openapi_python_client\
- && pytest --cov openapi_python_client tests\
+ && pytest --cov openapi_python_client tests --cov-report=term-missing\
```

Comment:
What's that do?

Reply:
When it shows the per-file coverage percentages in the report, it also shows which line numbers are missing next to it which is an absolute godsend